packages/indy-vdr/src/pool/IndyVdrPool.ts

```diff
@@ -61,6 +62,12 @@ export class IndyVdrPool {
       throw new IndyVdrError('Cannot connect to pool, already connected.')
```

Comment:
what if we just return if the pool is already defined? Then we don't have to store the archived pool, and we don't set it to undefined on close (until we fix indy-vdr)

Reply:
True, that's probably the easiest way around, thanks :sweat_smile: